server/src/utils/mixerConnections/LawoRubyConnection.ts

```diff
@@ -515,17 +520,24 @@ export class LawoRubyMixerConnection {
                 'Oops could not find node or function to update PFL state'
             )
 
-        this.emberConnection.invoke(
-            fn,
-            {
-                value: fader,
-                type: Model.ParameterType.String,
-            },
-            {
-                value: state.faders[0].fader[channelIndex].pflOn,
-                type: Model.ParameterType.Boolean,
+        try {
+            const { response } = await this.emberConnection.invoke(
+                fn,
+                {
+                    value: fader,
+                    type: Model.ParameterType.String,
+                },
+                {
+                    value: state.faders[0].fader[channelIndex].pflOn,
+                    type: Model.ParameterType.Boolean,
+                }
+            )
+            if (response) {
+                await response
             }

```

Comment:
Just a note: Since awaiting a literal will result in a resolved promise, this can be reduced to
```suggestion
            await response
```